site/en/setup.md

```diff
@@ -196,12 +203,21 @@ make all OCRD_MODULES="core ocrd_tesserocr ocrd_cis" # Will install only ocrd_te
 
 (Custom choices for `OCRD_MODULES` and other control variables (cf. `make help`) can also be made permanent by writing them into `local.mk`.)
 
-**Note: Never run `make all` as root unless you know *exactly* what you are doing!
+**Note:** Never run `make all` as root unless you know *exactly* what you are doing!
 
 Installation is incremental, i.e. failed/interrupted attempts can be continued, and modules can be installed one at a time as needed.
 
 Running `make` will also take care of cloning and updating all required submodules.
 
+Especially running `make all` will take a while. In the end, it should say that the last processor was installed successfully.
+
+Having installed `ocrd_all` successfully, `ocrd --version` should give you the current version of `ocrd core`
```

Comment:
`ocrd core` -> [OCR-D/core](https://github.com/OCR-D/core)

Current version is 2.13.2 btw :)

Reply:
really? I just updated this morning...

Reply:
yep. it's not in ocrd_all yet tho